main.go

```diff
@@ -23,15 +24,16 @@ import (
 var (
 	success = color.New(color.FgGreen)
 	fail    = color.New(color.FgHiRed)
-	ignore = true
+	ignore = flag.Bool("ignore",false,"ignore packages with no test files")
```

Comment:
I don't believe `-ignore` is a descriptive enough flag name.

Reply:
What is your suggestion?

Reply:
I changed the name to -skipnotest